### ISSUE TITLE
Hello bank! - France

### ISSUE THREAD

**Issue by d4rklynk** [PrivSec-dev/banking-apps-compat-report#163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Hello bank!

### Link to app

https://play.google.com/store/apps/details?id=com.bnpp.hellobank&gl=US

### App version

v3.28.0

### Country of the app

France

### Build Number

TP1A.221005.003.2022100300

### Device

- [X] Pixel 6a
- [ ] Pixel 6 Pro
- [ ] Pixel 6
- [ ] Pixel 5a
- [ ] Pixel 5
- [ ] Pixel 4a (5G)
- [ ] Pixel 4a
- [ ] Pixel 4 XL
- [ ] Pixel 4
- [ ] Pixel 3a XL
- [ ] Pixel 3a
- [ ] Pixel 3 XL
- [ ] Pixel 3

### Google Play installed

- [X] Installed
- [X] Not installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### SafetyNet Enforcement

- [ ] Enforced
- [X] Not enforced
- [ ] Unsure

### Native code debugging

- [X] Enabled
- [ ] Disabled

### Stock OS compatibility

- [X] Works
- [ ] Does not work
- [ ] Not tested

### Profile app tested in

- [X] Main user profile
- [ ] Secondary user profile
- [X] I used a work profile device manager app (Please mention more details about it down below like the app)

### Description of the app's functionality

Everything works. 

It also works **without** Google Play but with annoying pop-up "Google Play Services missing" sometimes, that you can **bypass** by clicking elsewhere.

**Comment by akc3n** [comment]
Thank you @d4rklynk

**Comment by d4rklynk** [comment]
### Update 2023.04.07
TL;DR : just updating the versions

- I updated the BUILD number of GrapheneOS and the app version
- The statements made in this report are still correct
- It sill works w/ or w/o Google Play Services

**Comment by jlsnt** [comment]
"Clé digitale" is not working in the app. Getting an error "16". Customer service telling it's due to the "non stock" OS running on the phone.

**Comment by d4rklynk** [comment]
> "Clé digitale" is not working in the app. Getting an error "16". Customer service telling it's due to the "non stock" OS running on the phone.

Have you unlocked Verified Boot ?

**Comment by d4rklynk** [comment]
Because I still have the app, it works like a charm

**Comment by jlsnt** [comment]
> Have you unlocked Verified Boot ? Double check if it's [enabled](https://grapheneos.org/install/web#locking-the-bootloader). Also, I'm not sure it's related, but check if you have disabled [OEM unlocking](https://grapheneos.org/install/web#disabling-oem-unlocking)

Hello,

I used the web installer when I installed GOS a few months ago, therefore I believe the bootloader was locked as this action is part of the (automated) process. The only thing I can verify right now is whether "OEM unlocking" is disabled, which is the case (it is disabled).

**Comment by d4rklynk** [comment]
> Hello,
> 
> I used the web installer when I installed GOS a few months ago, therefore I believe the bootloader was locked as this action is part of the (automated) process. The only thing I can verify right now is whether "OEM unlocking" is disabled, which is the case (it is disabled).

Hm, weird, I know Hello Bank can be a bit tricky with their "clé digitale". I would like to reinstall to check that, but I'm afraid I will not be able to use it anymore x)

**Comment by spring-onion** [comment]
Does this one still work?

**Comment by jlsnt** [comment]
Hi,

Did you mean "still does **not** work" ? :)
I'm now getting an error 91 when trying to enroll my phone for their "digital key".

**Comment by spring-onion** [comment]
Yeah that's what I'm wondering, there's conflicting information in here! What functionality is locked behind this digital key thing?

**Comment by jlsnt** [comment]
Gave it another try this evening : still not working, rejected with "error 91". Sent a message to the bank asking to explain this. Expecting the usual answer "we don't accept phones with alternate OS for security reasons".

**Comment by jlsnt** [comment]
Follow-up... Their answer at the time was as expected (a bad one).

However last month or so, I attempted once again to enroll for the "clé digitale". This worked and I was able to validate a few online purchases from the app (replacing obsolete/insecure SMS)... until it stopped working mid-december.

At this point the app was indicating "clé digitale" was enrolled on "another mobile" (Parameters > Clé digitale). It was offering to re-enroll but that action was failing with some error (can't remember the error code).

It's now back to a fully working state, without any action on my side (the app was updated on Dec 23rd though).

**Comment by 002gRTEah3mRYH** [comment]
Hello, did someone manage to make "Clé Digitale" working in Private Space ?

They send links in SMS main profile, but obviously the SMS doesn't reach the app in Private Space.

_I know Fortuneo was similar, but using email, so installing the mail app with the banking app inside Private Space makes it work.
But with HelloBank using SMS.._

**EDIT** : ok, I just installed Session (messaging app) to share the link from main profile to private space, **now working**
Using  aurora store btw

**Comment by pled** [comment]
Hello,

Has anyone been installing this app recently ? Is it still possible to install latest version on GOS phone ? 
Thanks.

**Comment by 002gRTEah3mRYH** [comment]
@pled 2 weeks and 1 week ago

Working in main profile and private space for me

(Clé digitale included (2FA stuff))
